ming_sim/action_materialize.py

```
@@ -1501,7 +1890,9 @@ def require_materializable_xiexang_payload(
     )
     body = str(text or "").strip()
     if not body:
-        raise ValueError("协饷旨意缺少正文（不猜散文）")
+        raise DecreeMaterializationValidationError(
+            "协饷旨意缺少正文（不猜散文）", failed_fields=("purpose",),
```

Ruff (0.16.3)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)